<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>paraYatir</class>
 <widget class="QWidget" name="paraYatir">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>424</width>
    <height>550</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QComboBox" name="cmParaTuru">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>250</y>
     <width>103</width>
     <height>41</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>-1</number>
   </property>
   <item>
    <property name="text">
     <string>Dolar</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Tl</string>
    </property>
   </item>
  </widget>
  <widget class="QLineEdit" name="lnParaMiktari">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>260</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>201</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:18pt; font-weight:700; color:#0433ff;&quot;&gt;Para Yatirma İşlemi&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>190</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Para Türü&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>190</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Para Tutarı&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbParaYatir">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>400</y>
     <width>100</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Para Yatir</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbGeri">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>510</y>
     <width>100</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Geri</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lnTC">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>135</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>135</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>TC Numarası</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelDurum">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>300</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
